package.xml

```diff
@@ -131,6 +131,16 @@
         <configureoption name="with-mcrypt" default="autodetect" prompt="libmcrypt prefix?"/>
     </extsrcrelease>
     <changelog>
+        <release>
+            <date>2018-12-23</date>
+            <version>
+                <release>1.0.2</release>
+                <api>1.0.0</api>
+            </version>
+            <notes>
+                - Changed maximum version to 7.3.x.
+            </notes>
+        </release>
```

Comment:
I believe this change is only made after the release.

Reply:
I reverted package.xml.